src/processing/spot.rs

```diff
@@ -52,6 +53,14 @@ pub async fn process_data_by_pair(
                 .first(&mut conn)
                 .await?
         }
+        NetworkName::PragmaDevnet => {
+            testnet_dsl::spot_entry
+                .filter(testnet_dsl::network.eq("pragma-devnet"))
+                .filter(testnet_dsl::pair_id.eq(pair.clone()))
+                .order(testnet_dsl::block_timestamp.desc())
```

Comment:
testnet_dsl ?

Reply:
woaaa missed all of those, didn't spaw that we updated them. Corrected!